dotnet/src/Connectors/Connectors.Memory.AzureAISearch/AzureAISearchMemoryRecord.cs

```diff
@@ -11,6 +12,7 @@ namespace Microsoft.SemanticKernel.Connectors.AzureAISearch;
 /// Azure AI Search record and index definition.
 /// Note: once defined, index cannot be modified.
 /// </summary>
+[Experimental("SKEXP0020")]
```

Comment:
Do we need to do this for internal classes?

Reply:
I'm adding experimental to internal classes as well, since I don't want them accidentally used by us.